modell/src/main/kotlin/no/nav/tilbakekreving/behandling/saksbehandling/vilkårsvurdering/Skyldgrad.kt

```diff
@@ -75,6 +75,61 @@ sealed interface Skyldgrad : ForårsaketAvBruker.Ja {
         }
     }
 
+    class UaktsomtUnder4xRettsgebyrUnnlates(
+        override val begrunnelse: String,
+        override val feilaktigeEllerMangelfulleOpplysninger: FeilaktigEllerMangelfull,
+        override val begrunnelseAktsomhet: String,
+    ) : Skyldgrad {
+        override fun renter() = false
+
+        override fun reduksjon(): Reduksjon = Reduksjon.IngenTilbakekreving()
+
+        override fun vurderingstype(): Aktsomhet = Aktsomhet.SIMPEL_UAKTSOMHET
+
+        override fun tilFrontendDto(): VurdertVilkårsvurderingsresultatDto? {
+            return VurdertVilkårsvurderingsresultatDto(
+                vilkårsvurderingsresultat = feilaktigeEllerMangelfulleOpplysninger.vilkårsvurderingsresultat,
+                aktsomhet = VurdertAktsomhetDto(
+                    aktsomhet = Aktsomhet.SIMPEL_UAKTSOMHET,
+                    ileggRenter = renter(),
+                    andelTilbakekreves = reduksjon().andel,
+                    beløpTilbakekreves = null,
+                    begrunnelse = begrunnelse,
+                    særligeGrunner = null,
+                    tilbakekrevSmåbeløp = false,
+                ),
+            )
+        }
+
+        override fun tilEntity(): AktsomhetsvurderingEntity {
+            return AktsomhetsvurderingEntity(
+                vurderingType = VurderingType.FORÅRSAKET_AV_BRUKER,
+                begrunnelse = begrunnelse,
+                beløpIBehold = null,
+                aktsomhet = VurdertAktsomhetEntity(
+                    aktsomhetType = AktsomhetType.SIMPEL_UAKTSOMHET_UNNLATES,
```

Comment:
Samme her

Reply:
Egentlig er det uaktsomt som burde ha en `KanUnnlates4xRettsgebyr` i stedet for SærligeGrunner instanse